aesara/tensor/random/basic.py

```diff
@@ -1161,20 +1535,51 @@ def rng_fn_scipy(cls, rng, alpha, p, lambd, size):
 
 
 class MultinomialRV(RandomVariable):
-    """A Multinomial random variable type.
+    r"""A multinomial discrete random variable.
+
+    The probability mass function of `multinomial` in terms of the number
+    of experiments :math:`n` and the probabilities :math:`p_1, \dots, p_k`
+    of the :math:`k` different possible outcomes is:
+
+
+    .. math::
+
+        f(x_1,\dots,x_k; n, p_1, \dots, p_k) = \frac{n!}{x_1! \dots x_k!} \prod_{i=1}^k x_i^{p_i}
+
+
+    where :math:`n>0` and :math:`\sum_{i=1}^k p_i = 1`.
 
     Notes
     -----
     The length of the support dimension is determined by the last
     dimension in the *second* parameter (i.e.  the probabilities vector).
-    """
 
+    """
     name = "multinomial"
     ndim_supp = 1
     ndims_params = [0, 1]
     dtype = "int64"
     _print_name = ("MN", "\\operatorname{MN}")
 
+    def __call__(self, n, p, size=None, **kwargs):
+        r"""Draw samples from a discrete multinomial distribution.
+
+        Parameters
+        ----------
+        n
+            Number of experiments :math:`n`. Must be a positive integer.
+        p
+            Probabilities of each of the different outcomes. The length of its
+            last dimension determines the number of dimensions of the support.
+        size
+            Sample shape. If the given size is, e.g. `(m, n, k)` then `m * n * k`
+            independent, identically distributed random variables are
+            returned if `p` broadcasts to `(m, n, k, p.shape[-1:])`, raises an error
+            otherwise. If the size is `None` (default) then `np.broadcast(n, p)` samples
+            are drawn.
```

Comment:
```suggestion
            independent, identically distributed samples are
            returned. If size is `None` (default), then a single sample
            is returned.
```

Reply:
If size is `None`, then `np.broadcast(n, p).size` samples are indeed drawn.

```python
import aesara.tensor as at
import numpy as np

srng = at.random.RandomStream(0)
n = np.array([10, 9, 11])
p = np.array([[0.5, 0.5], [.1, .9], [.2, .8]])
rv = srng.multinomial(n, p)

print(np.shape(np.broadcast(n, p)))
# (3, 2)
print(rv.eval().shape)
# (3, 2)
```